internal/cmd/users.go

```diff
@@ -348,7 +348,13 @@ func updateUser(cli *CLI, name string) error {
 }
 
 func getUserByNameOrID(client *api.Client, name string) (*api.User, error) {
-	users, err := client.ListUsers(api.ListUsersRequest{Name: name})
+	showSystem := false
+
+	if name == "connector" {
```

Comment:
Did you want to use `models.InternalInfraConnectorIdentityName` here as well?

Reply:
We use the name directly in `cmd` to avoid importing the models.